### PR TITLE
Adding static links to Dr. CI bot comment

### PR DESCRIPTION
**Pull request by swang392**
**Overview:** Edited `formDrciComment` function to include static helpful links -- hud and office hours link

**Test Plan:** modified test case to check that both links are there. Example output of the comment:
Screen Shot 2022-06-24 at 3 14 55 PM

### CODE REVIEW

**Comment by swang392** [comment]
> Is there a way to view what an example comment might look like?

I'm not sure, I was looking at other bot comments and I think it follows markdown format so thats what I used

File: torchci/lib/bot/drciBot.ts [hunk 1]
```diff
@@ -23,8 +24,12 @@ async function getDrciComment(
   return {id: 0, body: ""};
 }
 
-export function formDrciComment(): string {
-  let body = "hello there!"
+export function formDrciComment(prNum: number): string {
+  const hudUrl = "hud.pytorch.org/pr/" + prNum;
+  let body = "<h1>Helpful Links</h1>\n";
```
Comment (zengk95) [inline torchci/lib/bot/drciBot.ts:29]: Did not know that it supported html. I think the main convention is to just use markdown though so you can maybe try changing it to 
`# Helpful Links` or something

File: torchci/lib/bot/drciBot.ts [hunk 2]
```diff
@@ -23,8 +24,12 @@ async function getDrciComment(
   return {id: 0, body: ""};
 }
 
-export function formDrciComment(): string {
-  let body = "hello there!"
+export function formDrciComment(prNum: number): string {
+  const hudUrl = "hud.pytorch.org/pr/" + prNum;
```
Comment (zengk95) [inline torchci/lib/bot/drciBot.ts:28]: it's usually better to do something like 
` `hud.pytorch.org/pr/${prNum}` ` for better readability when string formatting. ( same for the other stuff on line 30 and 31).

Comment (zengk95) [reply torchci/lib/bot/drciBot.ts:28]: also maybe move this constant up.

File: torchci/test/drciBot.test.ts [hunk 3]
```diff
@@ -30,7 +30,10 @@ describe("verify-drci-functionality", () => {
       })
       .reply(200)
       .post("/repos/zhouzhuojie/gha-ci-playground/issues/31/comments", (body) => {
-        expect(body).toMatchObject({ body: "<!-- drci-comment-start -->hello there!<!-- drci-comment-end -->" });
+        const comment = body.body;
+        expect(comment.includes("<!-- drci-comment-start -->")).toBeTruthy();
```
Comment (zengk95) [inline torchci/test/drciBot.test.ts:34]: you can maybe export the const drciCommentStart and then import it here to do the comparison. Might be a bit easier to maintain.

**Review by zengk95 — APPROVED** [review]
lgtm

Only thing is that I feel like there should be bullet points on the links but can be added later I guess.

**Pull request merged**